README.md

```diff
@@ -1,4 +1,4 @@
-<img src="/docs/img/logo.jpg" width="500px" alt="kops logo">
+<img src="/docs/img/logo.jpg" width="500px" alt="kops logo"> 
```

Comment:
Don't really see this as a change that should be in this PR

Reply:
i think it was to trigger ci again as it flaked on a test.